speed_to_erpm_gain: 4082
speed_to_erpm_offset: 0.7
tachometer_tickes_to_meters_gain: 0.00255
#max_servo_speed: 3.2
#servo_smoother_rate: 75.0
max_acceleration: 2.5
throttle_smoother_rate: 75.0
steering_angle_to_servo_gain: -1.2135
steering_angle_to_servo_offset: 0.5304
vesc_to_odom/publish_tf: true
wheelbase: 0.504
wheelradius: 0.1
